examples/Format_Features/functions_in_rhs.yml

```diff
@@ -0,0 +1,13 @@
+odes:
+    - stateId: x_1
+      rightHandSide: f(x_2, 1, 2)
+      initialValue: 1
+
+    - stateId: x_2
+      rightHandSide: f(x_1, -1, 0)
+      initialValue: 1
+
+functions:
+    - functionId: f
+      arguments: x, a, b
```

Comment:
Could use an array here to specify the individual arguments, to avoid parsing a string

Reply:
But this would elongate/complicate the yaml definition, wouldn't it?

Reply:
Yes, my mistake, just saw that these arguments are directly substituted into the function string as is [1], without need for parsing. So, fine as is :+1: Even if parsing was required, the current format might be preferable anyway.

[1] https://github.com/yaml2sbml-dev/yaml2sbml/blob/master/yaml2sbml/yaml2sbml.py#L334